packages/camera/android/build.gradle

```diff
@@ -1,6 +1,6 @@
 group 'io.flutter.plugins.camera'
 version '1.0-SNAPSHOT'
-def args = ["-Xlint:deprecation","-Xlint:unchecked","-Werror"]
+def args = ["-Xlint:deprecation","-Xlint:unchecked"]
```

Comment:
Why do we need to remove `-Werror`? are we hitting warnings we cannot fix?

Reply:
I was running into some warning when running the (already existing) `DartMessengerTest` locally. I shouldn't have checked this in.